csharpcore/GildedRoseTests/GildedRoseTest.cs

```diff
@@ -1,18 +1,65 @@
-﻿using Xunit;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using GildedRoseKata;
+using NUnit.Framework;
 
 namespace GildedRoseTests
 {
+    [TestFixture]
     public class GildedRoseTest
     {
-        [Fact]
-        public void foo()
+        [TestFixture]
+        public class UpdateQuality
         {
-            IList<Item> Items = new List<Item> { new Item { Name = "foo", SellIn = 0, Quality = 0 } };
-            GildedRose app = new GildedRose(Items);
-            app.UpdateQuality();
-            Assert.Equal("fixme", Items[0].Name);
+            [TestFixture]

```

Comment:
I like to use PR to discuss about the code.
Even good things or just to describe what was done.

So [TestFixture] can be used to group some related test that share a common input.
It makes a Wonderfull tree when you run your test